criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
```

Comment:
Maybe "necessary" instead?

Reply:
Just chatted about this with Justin and the CNCF baseline WG— the solution to this will be to add a glossary/lexicon that defines our terms. When we create that document, we can also come back through to make sure it has successfully clarified lines like this.